core-android/src/test/java/com/uber/sdk/android/core/auth/LoginButtonTest.java

```diff
@@ -153,7 +154,7 @@ public void testOnActivityResult_shouldCascadeLoginManager() {
 
         loginButton.onActivityResult(LoginManager.REQUEST_CODE_LOGIN_DEFAULT, 1, intent);
 
-        verify(loginManager).onActivityResult(eq(activity), eq(LoginManager.REQUEST_CODE_LOGIN_DEFAULT), eq(1),
+        verify(loginManager).onActivityResult(ArgumentMatchers.eq(activity), eq(LoginManager.REQUEST_CODE_LOGIN_DEFAULT), eq(1),
                 eq(intent));
```

Comment:
Those changes here seem No-op -- and I actually find the plain `eq` more readable